src/cr/cube/cube.py

```diff
@@ -312,6 +316,13 @@ def is_weighted(self):
         """True if cube response contains weighted data."""
         return self._measures.is_weighted
 
+    @lazyproperty
+    def means_subvariables(self):
+        """List of subvar names for numeric arrays, None otherwise."""
+        if not self.has_means:
+            return None
```

Comment:
This is what I was referring to. I know that it's the API method (property), and that it's not the same, but we should avoid confusing an unsuspecting reader of the code :)

Reply:
ok